webapp/platform/client/src/client4.ts

```diff
@@ -756,6 +756,23 @@ export default class Client4 {
         return profile;
     };
 
+    loginWithDesktopToken = async (token: string) => {
+        const body: any = {
+            token,
+            deviceId: '',
+        };
+
+        const {
+            data: profile,
+            headers,
+        } = await this.doFetchWithResponse<UserProfile>(
```

Comment:
Should we still set the `Token` header here line we do above? If not, this could be changed to use `doFetch` instead of `doFetchWithResponse`.

Also, we might want to add a call to `this.trackEvent` since we have that for the other login methods here